gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -572,6 +572,19 @@ case class TransformPostOverrides(session: SparkSession, isAdaptiveContext: Bool
       p.withNewChildren(children)
   }
 
+  def nativeSupportedPlan(child: SparkPlan): Boolean = {
+    if (child.isInstanceOf[TransformSupport]) {
```

Comment:
There are some native operators that not extending `TransformSupport`, e.g., `ColumnarShuffleExchangeExec`. Recently, we have a common trait added, named `GlutenPlan`. Maybe, we should use it for checking. Thanks!